archunit/src/main/java/com/tngtech/archunit/library/GeneralCodingRules.java

```diff
@@ -151,4 +154,28 @@ private GeneralCodingRules() {
     @PublicAPI(usage = ACCESS)
     public static final ArchRule NO_CLASSES_SHOULD_USE_JODATIME =
             noClasses().should(USE_JODATIME).because("modern Java projects use the [java.time] API instead");
+
+    /**
+     * For information about checking this condition, refer to {@link GeneralCodingRules}.
+     */
+    @PublicAPI(usage = ACCESS)
+    public static final ArchCondition<JavaField> USE_FIELD_INJECTION = useFieldInjection().as("use field injection");
```

Comment:
Shouldn't overwriting the description happen within a factory method of a condition?